### ISSUE TITLE
Missing padding between radio buttons in Form questions

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1642]
The radio buttons in the form questions are grouped very close together. This might pose an issue for people with unsteady hands or low vision. [The minimum target size is 24 by 24 pixels (with some exceptions).](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)
![image](https://github.com/user-attachments/assets/29301447-0049-4010-ae5f-ea3d28ed2c4d)
